src/tools/publish_toolstate.py

```diff
@@ -85,8 +116,11 @@ def update_latest(
                         .format(tool, os, old, new, MAINTAINERS.get(tool))
                 elif new < old:
                     changed = True
-                    message += '💔 {} on {}: {} → {} (cc {}, @rust-lang/infra).\n' \
-                        .format(tool, os, old, new, MAINTAINERS.get(tool))
+                    title = '💔 {} on {}: {} → {}' \
+                        .format(tool, os, old, new)
+                    message += '{} (cc {}, @rust-lang/infra).\n' \
+                        .format(title, MAINTAINERS.get(tool))
+                    issue(title, tool, MAINTAINERS.get(tool), relevant_pr_number, relevant_pr_user)
```

Comment:
If a PR breaks 3 tools on both platforms this will open 6 issues. I think this too much. Could you add the issue check at the end so only 1 issue will be opened?